src/dom/DOMPipe.ts

```diff
@@ -97,18 +106,8 @@ export class DOMPipe implements RenderPipe<DOMContainer>
             return;
         }
 
-        const canvas = this._renderer.view.canvas as HTMLCanvasElement;
-
-        if (this._domElement.parentNode !== canvas.parentNode)
-        {
-            canvas.parentNode?.appendChild(this._domElement);
-        }
-
-        const sx = (parseFloat(canvas.style.width) / canvas.width) * this._renderer.resolution;
-        const sy = (parseFloat(canvas.style.height) / canvas.height) * this._renderer.resolution;
-        // scale according to the canvas scale and translate
-
-        this._domElement.style.transform = `translate(${canvas.offsetLeft}px, ${canvas.offsetTop}px) scale(${sx}, ${sy})`;
+        // Ensure the canvas is set
+        this._canvasTransformSync.canvas = this._renderer.view.canvas as HTMLCanvasElement;
```

Comment:
I think we can simplify this to leveraging `init` hook in the systems 

 we can create `CanvasTransformSync`

```
  this._canvasTransformSync = new CanvasTransformSync({
            domElement: this._domElement,
            renderer: this._renderer, // canvas / view will be ready
  });
```

that way `CanvasTransformSync` does not need an init function and can be set up in the constructor simplifying the code flow.